scripts/services/getConfig.ts

```diff
@@ -1,238 +1,345 @@
                   "pattern-description":
                     "Must be either a domain name, or an IPv4 or IPv6 address. Do not include protocol scheme (eg 'http://') or port.",
                 },
-                "port": {
-                  "type": "number",
-                  "nullable": true,
-                  "name": "Port",
-                  "description":
+                port: {
+                  type: "number",
+                  nullable: true,
+                  name: "Port",
+                  description:
                     "TCP Port that peer is listening on for inbound p2p connections.",
-                  "range": "[0,65535]",
-                  "integral": true,
-                  "default": 18080,
+                  range: "[0,65535]",
+                  integral: true,
+                  default: 18080,
                 },
-                "prioritynode": {
-                  "type": "boolean",
-                  "name": "Priority Node",
-                  "description":
+                prioritynode: {
+                  type: "boolean",
+                  name: "Priority Node",
+                  description:
                     "Attempt to stay perpetually connected to this peer.",
-                  "default": false,
+                  default: false,
                 },
               },
             },
           },
         },
       },
-      "pruning": {
-        "type": "boolean",
-        "name": "Pruning",
-        "description":
-          "Blockchain pruning in Monero prunes proof data from transactions after verification but before storage.  This saves roughly 2/3s of disk space.  The drawback of pruning is that you will contribute less to Monero P2P network in terms of helping new nodes doing IBD. If enabled, your node will still relay new transactions and blocks.",
-        "default": false,
+      tor: {
+        type: "object",
+        name: "Tor",
+        description: "Tor anonymity network settings",
+        spec: {
+          toronly: {
+            name: "Tor only",
+            description:
+              "Only communicate with Monero nodes via Tor.  This is more private, but can be slower, especially during initial sync.",
+            type: "boolean",
+            default: true,
+          },
+          rpcban: {
+            type: "boolean",
+            name: "Ban misbehaving RPC clients",
+            description:
+              "This switch determines whether or not to ban hosts that generate RPC errors.  Disabling this may help to prevent monerod from banning traffic originating from the Tor daemon.  This can be useful to disable when using Tor-only mode, where every connection inbound to the onion's RPC appears to be from same remote host.",
```

Comment:
```suggestion
              "Ban hosts that generate RPC errors.  Leaving disabled may help to prevent monerod from banning traffic originating from the Tor daemon.  This is useful in Tor-only mode, where every connection inbound to the onion's RPC appears to be from same remote host.",
```